web/client-api/src/main/java/io/deephaven/web/client/api/JsLayoutHints.java

```diff
@@ -48,6 +85,15 @@ public JsLayoutHints parse(String hints) {
             frozenColumns = JsObject.freeze(freezeStr.split(","));
         }
 
+        final String groupsStr = options.get("groups");
+        if (groupsStr != null && !groupsStr.isEmpty()) {
```

Comment:
double checking: this means that the `columnGroups` will be null and not empty if groupsStr is either absent, provided as null, or an empty string. wouldn't it be nicer to work with if it was never null, but instead was empty?

Reply:
I'm just mimicking the logic from the other keys. It could be null b/c the layout hint builder does not add elements to the layout hints string if they aren't specified (i.e. no `columnGroups=""`)